TestResultSummaryService/parsers/BenchmarkParser.js

```diff
@@ -90,45 +91,40 @@ class BenchmarkParser extends Parser {
-
-                curBenchmarkNamedRouted = BenchmarkMetricRouter[curBenchmarkName][curBenchmarkVariant];
+            	
+                curBench = BenchmarkMetric[BenchmarkMetricRouter[curBenchmarkName][curBenchmarkVariant]];
 
                 // Parse metric values
                 curTestData["metrics"] = [];
-                for ( let i = 0; i < BenchmarkMetric[curBenchmarkNamedRouted]["metrics"].length; i++ ) {
-                    curMetric = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].name;
-                    curSearchString = curItr.value;
+                for ( let i = 0; i < curBench["metrics"].length; i++ ) {
 
+                    curMetric = curBench["metrics"][i].name;
+                    curSearchString = curItr.value;
                     // metric values will be found within the result of the outer regex
-                    if (BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].outerRegex !== undefined) {
-                        if ( ( curRegexResult = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].outerRegex.exec( curSearchString ) ) !== null ) {
+                    if (curBench["metrics"][i].outerRegex !== undefined) {
+                        if ( ( curRegexResult = curBench["metrics"][i].outerRegex.exec( curSearchString ) ) !== null ) {
                             curSearchString = curRegexResult[1];
                         } else {
                             // No result for outer regex, skip this metric
                             continue;
                         }
                     }
-
-                    // multiple metric values are present
-                    if (BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].regexRepeat === true) {
-                        curMetricValue = [];
-                        curRegexResult = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].regex.exec( curSearchString )
-
-                        while (curRegexResult !== null ) {
-                            curMetricValue.push(parseFloat(curRegexResult[1]));
-                            curRegexResult = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].regex.exec( curSearchString );
-                        }
-
-                    // single metric value
-                    } else {
-                        if ( ( curRegexResult = BenchmarkMetric[curBenchmarkNamedRouted]["metrics"][i].regex.exec( curSearchString ) ) !== null ) {
-                            curMetricValue = [parseFloat(curRegexResult[1])];
-                        }
-                    }
-
-                    curTestData["metrics"].push({name: curMetric, value: curMetricValue})
-                }
-
+                    
+                    curMetricValue = [];
+                    /*	Parse all values for single metric from result to an array
+                     *  e.g Liberty Startup =>
+                     *  curRegexResult = ['startup time in ms 32',32,'startup time in ms 41',41, x 6] 
```

Comment:
Let's add another example for throughput as requested earlier. Please review all the previous comments before updating the PR in order to avoid reviewing the same issue again. Thanks!